src/Build.UnitTests/Evaluation/Expander_Tests.cs

```diff
@@ -2010,13 +2010,16 @@ public void PropertyFunctionNullArgument()
         public void PropertyFunctionNullReturn()
         {
             PropertyDictionary<ProjectPropertyInstance> pg = new PropertyDictionary<ProjectPropertyInstance>();
-            pg.Set(ProjectPropertyInstance.Create("SomeStuff", "This IS SOME STUff"));
 
             Expander<ProjectPropertyInstance, ProjectItemInstance> expander = new Expander<ProjectPropertyInstance, ProjectItemInstance>(pg, FileSystems.Default);
 
-            string result = expander.ExpandIntoStringLeaveEscaped("$([System.Convert]::ChangeType(,$(SomeStuff.GetType())))", ExpanderOptions.ExpandProperties, MockElementLocation.Instance);
```

Comment:
Was this just a fancy way to get `null` before, and you're using the easier approach now?

Reply:
It was not evaluating to `null` before so not hitting the intended code path. Presumably because the `(,` argument omission makes us pass an empty string.